{{cookiecutter.project_slug}}/Makefile

```diff
@@ -122,15 +122,15 @@ training:  ### Run training job
 		$(TRAINING_COMMAND)
 
 .PHONY: kill-training
-kill-training:  ### Stop training job
+kill-training:  ### Terminate a training job
 	neuro kill $(TRAINING_NAME)
 
 .PHONY: connect-training
-connect-training:  ### Execute shell to the training job
+connect-training:  ### Connect with a shell to a training job
 	neuro exec --no-key-check $(TRAINING_NAME) bash
 
 .PHONY: jupyter
-jupyter: upload-code upload-notebooks ### Run jupyter job
+jupyter: upload-code upload-notebooks ### Run a job with Jupyter notebooks and open UI in your default browser
```

Comment:
- "with Jupyter notebooks` -- many notebooks?
- "in your default browser" -- minor: I personally prefer computer programs not to violate subordination and not to call me "you". So I would change it to "open UI in the default browser" (system's default browser)

Reply:
note, the second comment is very minor and reflects my personal feeling, which, I'm sure, is not shared with many other engineers